### PR TITLE
Use absolute urls

### PR DESCRIPTION
**Pull request by adborden**
When sharing a url that includes the user/pass, Chrome blocks some assets in an
attempt to hide the creds. By using absolute urls, we can use a link with
user/pass to auth the user and all assets won't leak any information.

### CODE REVIEW

File: prototypes/form-wizard/_config.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ title: FOIA request prototype
 description: > # this means to ignore newlines until "baseurl:"
   A prototype exploring how the public might submit a FOIA request through
   a single national portal.
-baseurl: "" # the subpath of your site, e.g. /blog
+baseurl: "https://foia-form-wizard.app.cloud.gov" # the subpath of your site, e.g. /blog
```
Comment (pkarman) [inline prototypes/form-wizard/_config.yml]: will this break things when hosted locally (as in dev)?

Comment (adborden) [reply prototypes/form-wizard/_config.yml]: arg, yes.

**Review by pkarman — APPROVED** [review]
lgtm

**Comment by pkarman** [comment]
@adborden looks like you need a rebase

**Pull request merged**